burger/src/components/Main/index.js

```diff
@@ -29,6 +29,10 @@ const Main = () => {
       Object.values(priceIngridients).reduce((a, c) => a + c, 1) * 100
     ) / 100;
 
+  useEffect(() => {
+    setBurgerPrice(priceBurger);
+  });
```

Comment:
This `useEffect` willl run everytime the component with be triggered

```suggestion
  useEffect(() => {
    setBurgerPrice(priceBurger);
  }, []);
```